shellbase-slack/src/main/scala/com/sumologic/shellbase/slack/PostToSlackHelper.scala

```diff
@@ -26,7 +26,7 @@ import slack.models.{Attachment, Block}
 trait PostToSlackHelper {
   protected val slackState: SlackState
 
-  protected val username: String = System.getProperty("user.name", "unknown")
+  protected def username: String = System.getProperty("user.name", "unknown")
```

Comment:
I believe this is backward compatible, but not forward, which should be fine?  Might be worth a minor version bummp though.